addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -233,6 +233,12 @@ body.editor_enable {
         > * {
             pointer-events: none;
         }
+        // REMOVEME in master
+        // BS4 adds pointer-events: auto on this which breaks the editor.
+        // BS5 doesn't do that so this can safely be removed in master.
+        .stretched-link::after {
+            pointer-events: none;
+        }
```

Comment:
@detrouxdev There is a `pointer-events: none` rule just above :grimacing:

Reply:
It was to make it clear that only the .stretched-link class should be removed in master. But I just made the comment clearer instead. 👍🏻